android/src/main/java/com/horcrux/svg/DefsView.java

```diff
@@ -11,11 +11,12 @@
 import android.annotation.SuppressLint;
 import android.graphics.Canvas;
 import android.graphics.Paint;
+import android.graphics.Path;
 import android.view.View;
 import com.facebook.react.bridge.ReactContext;
 
 @SuppressLint("ViewConstructor")
-class DefsView extends DefinitionView {
+class DefsView extends RenderableView {
```

Comment:
DefinitionView is the exact fit for Defs; instead of changing it to Renderable, we should move the necessary functions from Renderable to Definition.